graph2tac/loader/hmodel.py

```diff
@@ -61,7 +60,7 @@ def args_decode(args,
 
 
 class Train:
-    def __init__(self, data_dir: Path, max_subgraph_size, with_context, shuffled, dry):
+    def __init__(self, data_dir: Path, output_dir: Path, max_subgraph_size, with_context, shuffled, dry):
```

Comment:
Good feature :-), (in my personal tests, I was previously moving `hmodel.sav` around the filesystem :-) )